hack/replace-nto.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
There's a lot of copy and paste from `test-upstream-tuned.sh` here.  I wonder if we should refactor `test-upstream-tuned.sh`, make some of the functions into a shell functions file. In any case, the duplication seems bad.

Reply:
Thanks @jmencak.
Now that I'm thinking of it, could we just get rid of the `test-upstream-tuned.sh` script?  It seems redundant now that we've decided to always track the tuned repo and stay updated (also my newly added script in this case does the same).

Reply:
`test-upstream-tuned.sh` has a clear purpose.  To deploy NTO image with a certain version of TuneD and run e2e tests on it.  FWICS, your script is basically a copy of `test-upstream-tuned.sh` (I'd expect that fact reflected in git history...) adds application of manifests. That's good.  But it drops the e2e test runs.  We can probably live with that and run the tests manually after the custom version is deployed.  So yeah, after you clean the script, we can probably do what you suggest.

Reply:
Yes, my script is pretty much copy and paste. I'd like to get it unified and have one script for both cases (as we don't need two separate scripts).
We can leave the e2e scripts, I personally thought this should be optional (maybe add documentation to hacking.md how to run them?). If you insist on keeping them, we can do it.

Reply:
I do not insist on having two scripts. The original script existed so that we have an easy way of testing new `TuneD` releases without adding extra commands. If you agree, we can e.g. "git mv test-upstream-tuned.sh deploy-custom-nto.sh" and then you can build on it.